Resources/Prototypes/Entities/Mobs/Customization/Markings/human_hair.yml

```diff
@@ -1377,11 +1377,17 @@
   sprites:
     - sprite: Mobs/Customization/human_hair.rsi
       state: longbow
-
 - type: marking
   id: HumanHairFrenchBraid
   bodyPart: Hair
   markingCategory: Hair
   sprites:
     - sprite: Mobs/Customization/human_hair.rsi
       state: frenchbraid
+- type: marking
```

Comment:
this edit needs to be moved into its own file, as does the locale file edit. we should not be altering upstream files at all anymore